### PR TITLE
Fix #925 added general get writemarker api

### LINKED ISSUE #925: Add new API latestwritemarker
`/latestwritemarker` to know the value in explorer app "Last time an allocation was written/read from." 
Add it as a RestHandller in [SmartContract](https://github.com/0chain/0chain/blob/9c78b97165efa9fc7ee415dcf04fca4d2e8cc03e/code/go/0chain.net/smartcontract/storagesc/sc.go#L48)

### PR DESCRIPTION
**Pull request by samtholiya**
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/0chain/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master BEFORE merging this PR  
  
Associated PRs (Link as appropriate):
...

### CODE REVIEW

**Comment by samtholiya** [comment]
Deleted this comment to avoid confusion

**Comment by samtholiya** [comment]
image

**Comment by samtholiya** [comment]
image

File: code/go/0chain.net/smartcontract/dbs/event/writemarker.go [hunk 1]
```diff
@@ -39,6 +41,14 @@ func (edb *EventDb) GetWriteMarker(txnID string) (*WriteMarker, error) {
 	return &wm, nil
 }
 
+func (edb *EventDb) GetWriteMarkers(offset, limit int, isDescending bool) ([]WriteMarker, error) {
+	var wm []WriteMarker
+	return wm, edb.Get().Debug().Model(&WriteMarker{}).Offset(offset).Limit(limit).Order(clause.OrderByColumn{
```
Comment (ssardana96) [inline code/go/0chain.net/smartcontract/dbs/event/writemarker.go]: Why is Debug here?

File: code/go/0chain.net/smartcontract/dbs/event/writemarker.go [hunk 2]
```diff
@@ -48,6 +58,11 @@ func (edb *EventDb) GetWriteMarkersForAllocationID(allocationID string) (*[]Writ
 	return &wms, result.Error
 }
 
+func (edb *EventDb) GetLatestWriteMarker() (WriteMarker, error) {
```
Comment (ssardana96) [inline code/go/0chain.net/smartcontract/dbs/event/writemarker.go]: is this getting used anywhere?

**Review by ssardana96 — CHANGES_REQUESTED** [review]

**Comment by ssardana96** [comment]
looks good. just couple of comments
Please discuss with @chauchausoup @shaktals on its integration.

**Comment by shaktals** [comment]
My opinion was that we did not need a specific API to get the last write or read marker, but just the possibility to order by timestamp and limit rows. Anyways, now that it is done, looks good to me.

File: code/go/0chain.net/smartcontract/dbs/event/writemarker.go (hunk 2 above)
Comment (samtholiya) [reply code/go/0chain.net/smartcontract/dbs/event/writemarker.go]: done

File: code/go/0chain.net/smartcontract/dbs/event/writemarker.go (hunk 1 above)
Comment (samtholiya) [reply code/go/0chain.net/smartcontract/dbs/event/writemarker.go]: done

**Comment by samtholiya** [comment]
@shaktals yes it is a general API delete the old comment to avoid confusion

**Pull request merged**